feature/platform/transceiver/tests/zr_tunable_parameters_test/zr_tunable_parameters_test.go

```diff
@@ -80,16 +80,21 @@ func Test400ZRTunableFrequency(t *testing.T) {
 		t.Run(tc.description, func(t *testing.T) {
 			for freq := tc.startFreq; freq <= tc.endFreq; freq += tc.freqStep {
 				t.Run(fmt.Sprintf("Freq: %v", freq), func(t *testing.T) {
-					gnmi.Replace(t, dut, gnmi.OC().Component(oc1).OpticalChannel().Config(), &oc.Component_OpticalChannel{
+					opticalChannel1Config := &oc.Component_OpticalChannel{
 						TargetOutputPower: ygot.Float64(tc.targetOutputPower),
 						Frequency:         ygot.Uint64(freq),
-						OperationalMode:   ygot.Uint16(dp16QAM),
-					})
-					gnmi.Replace(t, dut, gnmi.OC().Component(oc2).OpticalChannel().Config(), &oc.Component_OpticalChannel{
+					}
+					opticalChannel2Config := &oc.Component_OpticalChannel{
 						TargetOutputPower: ygot.Float64(tc.targetOutputPower),
 						Frequency:         ygot.Uint64(freq),
-						OperationalMode:   ygot.Uint16(dp16QAM),
-					})
+					}
+					if !deviations.OperationalModeUnsupported(dut) {
+						opticalChannel1Config.SetOperationalMode(dp16QAM)
+						opticalChannel2Config.SetOperationalMode(dp16QAM)
+					}
+					gnmi.Replace(t, dut, gnmi.OC().Component(oc1).OpticalChannel().Config(), opticalChannel1Config)
```

Comment:
Can you rewrite this slightly differently? This makes the default appear to not have a OperationalMode, but it gets added later on with a !deviation statement. When possible, I think it helps a lot with readability to flip that so the default is clear at the start.

Roughly this would then follow this format:

```
opticalChannel1Config := &oc.Component_OpticalChannel{
	TargetOutputPower: ygot.Float64(tc.targetOutputPower),
	Frequency:         ygot.Uint64(freq),
	OperationalMode:   ygot.Uint16(dp16QAM),
})

if deviations.OperationalModeUnsupported(dut) {
	opticalChannel1Config.OperationalMode = nil
}
```

Reply:
Changed the code accordingly, please check